src/rard/research/tests/views/test_fragment.py

```diff
@@ -492,3 +511,131 @@ def test_compare_orders(self):
             response.context_data["object_list"],
             [self.aftl1, self.aftl3, self.aftl2, self.aftl3],
         )
+
+
+class TestFragmentDuplicationView(TestCase):
+    def setUp(self):
+        self.cw = CitingWork.objects.create(title="citing work title")
+        self.ot = OriginalText.objects.create(
+            content="Original Text test",
+            apparatus_criticus_blank=False,
+            apparatus_criticus="here be the app crit",
+            object_id=111,
+            citing_work=self.cw,
+            content_type_id=12,
+            reference_order="reference order",
+        )
```

Comment:
apparatus_criticus isn't used, object_id and content_type (rather than content_type_id) shouldn't be set directly (I don't think), instead create the fragment first and set owner = fragment.